### DISCUSSION TITLE
Word Folders / Language Linking?

### DISCUSSION THREAD

**Discussion by infin-8-morphosis** [DraqueT/PolyGlot#1340] [Ideas]
So, I'm looking to convert my current Excel-based conlang into Polyglot, but the way I've been doing things is having a protolanuage which is converted into several final languages. Along with this, some words change meaning when they go from protolang to final language, like the verb 'grab' becoming 'steal'.

The way Polyglot seems to be, is that there is one language, and one language only. There is the etymology, but it only takes words from the current file. What I was going to do was sort of hackily make the protolanguage the main one and the others are conjugations of said language, and this would work... but then this cannot account for that change in definition, as far as I can tell...

Also, even without the whole multiple languages problem, correct me if I'm wrong but each word entry is isolated to itself, and has a singular part of speech. Is it not possible / more logical to make a word, and within its settings define it as a noun, verb, adjective, (and their definitions) instead of them being separate entries? For example, Boot (Parent Word) divided into Boot (Noun) Item of clothing, and Boot (Verb) to kick. Currently these need to be two separate entries, right?

This is probably too fundamental a change, but what I'd like is essentially the ability to have folders of words, so I can have a root word, and open it to set its part of speeches and corresponding meanings. I think this would be good for everyone, but for me specifically this would allow me (In an ideal perfect implementation) to click a proto word, see it in each final language, and then click again to see each final word as a noun, adjective, etc. Ideally cascading horizontally rather than vertically.

Alternatively, sort of, and maybe preferably but I think its more complicated than the idea of words as folders, but alternatively a way to 'link' words together, kind of like the etymology. To tell the system that one word is based on others and to convert or combine them automatically. For example compound words. Imagine an entry for 'fireplace'. Currently, if you change 'fire' to 'flame' the entry for 'fireplace' isnt going to become 'flameplace'. (I've been doing this in excel, with one greyed left box for automatic sound changes/compounding, and a black right box for manually tweaking awkward combos and such, as you can see below)

Here is an example of a single entry in my current setup. Actually its more complicated than I typed since I also have adjectives and verbs being converted back to nouns... But thats a whole other can of worms. 
![image](https://github.com/DraqueT/PolyGlot/assets/93674755/71e7f638-551e-4ebd-ba56-4712e81a8995)

In short, the individuality of the current lexicon is very limiting for me, and I'd need to use systems like the parts of speech and conjugation to do some of this, and it'd still be quite clunky. What I'd like is either (or both) a folder system for words so I can define nouns and verbs under their respective root, or a way to link a word to another word to form verbs, convert back to nouns, form compounds, etc.

Probably a big ask. But theres bits of this idea currently in the conjugation, etymology and lexical families (which seems close to the folder idea, in fact i'm not sure why its its own thing, it feels like something that should be baked into the words themselves), so maybe it would be possible to harmonise these currently disparate-feeling features into something like this?
